README.md

```diff
@@ -30,24 +30,25 @@ To use the Microsoft Graph Connect sample for AngularJS, you need the following:
 
 ## Register the application
 
-1. Sign into the [App Registration Portal](https://apps.dev.microsoft.com/) using either your personal or work or school account.
+1. Sign in to the [Azure Portal](https://portal.azure.com/) to register an application.
 
-2. Choose **Add an app**.
+1. If your account gives you access to more than one tenant, select your account in the top right corner, and set your portal session to the desired Azure AD tenant.
 
-3. Enter a name for the app, and choose **Create application**. 
-	
-   The registration page displays, listing the properties of your app.
+1. In the left-hand navigation pane, select the **Azure Active Directory** service, and then select **App registrations (Preview) > New registration**.
 
-4. Copy the Application Id. This is the unique identifier for your app. 
+1. When the **Register an application** page appears, enter a name for your application.
 
-5. Under **Platforms**, choose **Add Platform**.
```

Comment:
See my comments on [microsoftgraph/angular-connect-rest-sample]

Reply:
Thank you. Updated.